public/app/features/apiserver/types.ts

```diff
@@ -27,6 +27,8 @@ export interface ObjectMeta {
   generation?: number;
   // The first time this was saved
   creationTimestamp: string;
+  // The time this resource was marked for deletion
+  deletionTimestamp?: string;
```

Comment:
Is this really (optionally) available for all resources? 🤔 Is there somewhere else we could add it/isn't is in the generated endpoints file?